joomla/joomla-cms
# ACL Default attribute of access.xml

For now, when use ACL permission (by access.xml file for define rule), it's not support to set an default value for permission.

This PR is updated version of #7517 with some modification base on these comments from last PR. 

Don't have specific issue yet.
#### Summary of Changes

This PR for add an attribute "default" into an rule in access.xml file to set it default is "allow" or "denied", if default not specific, default rule will be "Inherit" as normal use. It's only effect when first edit permission rules and system will auto-select **allow** or **denied** base on "default" attribute in access.xml.

It's not has effect when this rule has been saved and open for edit.
#### Testing Instructions
- Add an attribute in "access.xml" of sample component (com_helloworld).
- Install this component.
- Go to configuration of this component and see the permission is auto selected follow value (1 => Allowed, 0 => Denied) of "default" attribute.
